src/components/ActionList/components/Section/tests/Section.test.tsx

```diff
@@ -77,7 +77,7 @@ describe('<Section />', () => {
   });
 
   it('passes the onActionAnyItem callback to Item', () => {
-    const callback = () => {};
+    const mockOnActionAnyItem = jest.fn();
```

Comment:
Should we use `spy` here?

Reply:
Per conversation in slack: My personal preference is for a more verbose name, but `spy` is the existing dominant convention. I'll use spy here to maintain existing convention, then we can update everything to more verbose names in a separate PR.